src/data/finset/nat_antidiagonal.lean

```diff
@@ -32,6 +32,35 @@ by simp [antidiagonal]
 @[simp] lemma antidiagonal_zero : antidiagonal 0 = {(0, 0)} :=
 rfl
 
+lemma antidiagonal_succ {n : ℕ} :
+  antidiagonal (n + 1) = insert (0,n + 1) ((antidiagonal n).map
```

Comment:
```suggestion
  antidiagonal (n + 1) = insert (0, n + 1) ((antidiagonal n).map
```

Reply:
I deleted some outdated review comments. Hopefully that won't lead to confusion...